documentation/intro-to-docs.md

```diff
@@ -0,0 +1,153 @@
+  alert's underlying causes, or at least rule out any common issues, with
+  the information provided in the document.
+* *Topical guides* provide a deep dive into how a specific system or
+  service works, with references to ADRs that illustrate how technical
+  decisions were made on the project. Readers should come away with a
+  strong understanding of how the system works, as well as how to gather
+  more information about the system and its state on their own.
+
+These aren't meant to be prescriptive; define your categories in a way
+that works best for your project. However, whatever you decide, the
+categories should be clearly defined and focused; notice the "what the
+reader should get from this document" part of the description -- if you
+cannot adequately summarize what a reader is supposed to get from the
+document in a single sentence, it is probably too widely scoped.
+
+### Keep your audience in mind always
+
+The most important thing when writing documentation is to remember that
+you are writing this *for* someone. It may be new engineers, it may
+be people on-call, it may be stakeholders in the C-suite, but whoever they
+are, you intend for *someone* to be reading this at some point in the
+future.
+
+If you aren't keeping this audience in mind throughout the process of writing
+your docs, then you are going to miss the mark. Writing a document intended
+for new engineers and assuming a ton of knowledge will severely limit its
+utility. Similarly, writing something intended for project leads and
+spending pages defining common terms they're already familiar with will
+probably cause them to lose patience and start to skim things over.
+
+### Avoid going overboard with graphics or video
+
+Often we'll be tempted to add lots of screenshots, diagrams, or other
+graphics to documentation to try and make things more clear. A well-placed
+graphic can help -- but too many will make the upkeep of your documentation
+too difficult. It is *much* more difficult to edit a graphic (and videos
+even more so) than to edit text, so use them only when they're worth the
+added cost in maintainability.
+
+In addition, adding lots of graphics will make it harder to read the
+text. If people cannot follow from one paragraph to the next (seeing them
+both on the screen at the same time), this is especially difficult. If
+you need to use large graphics, use thumbnails and link to the larger
+version to prevent them from being too distracting.
+
+### Keep individual documents short -- but not too short
+
+There's nothing worse than trying to find the one paragraph you need from
+a 20 page document. In order to avoid this, we should keep individual
+documents short and focused. Generally, anything more than 4 "pages" (ie,
+you would need to scroll four full screens to see the whole thing) is going
+to feel "long" to most readers -- if you go over this length, be aware that
+a lot of people will resist reading this documentation in a single sitting.
```

Comment:
Yea I think it's worth calling out that many people will skim the documentation so lists and tables are preferred over paragraphs of description.